If `sdl-config --libs` only outputs '-lSDL -lpthread', this code
will try to link with '-lSDL -lpthread' instead of '-lSDL' '-lpthread'

Patch by Rodolfo Boer.

http://bugs.gentoo.org/104221

--- SDL_Perl-2.1.2/make/lib/SDL/Build.pm
+++ SDL_Perl-2.1.2/make/lib/SDL/Build.pm
@@ -150,7 +151,7 @@
 			extra_linker_flags => 
 			[
 				@{ $links->{$subsystem}{paths} },
-				$sdl_link,
+				split / /, $sdl_link,
 				@{ $links->{$subsystem}{libs} },
 			],
 		},
